lib/rails/render/component/version.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Rails
+  module Render
+    module Component
+      VERSION = "5.0.1"
```

Comment:
Previous version we're following rails... I wonder if it's adequate to keep the same logic, if I am going to support all 5.x and 6.x ? Any opinion?